### DISCUSSION TITLE
[Discussion Thread] Shops

### DISCUSSION THREAD

**Discussion by KadTheHunter** [KadTheHunter/CityPlans#12] [General]
**Describe the Idea or Change below:**
What shops should be added and where.

**Is this Idea/Change Discussion related to an Idea Submission or other Issue?**
Issue #2, this is a remake of it.

**Comment by ParadiseMCPE** [comment]
- Miscellaneous Restaurants
- Food Carts
- Bakeries
- Apartment Complex
- Carnivals
- Boat Rentals
- Libraries, Museums
- Sports Arenas
- Prisons
- Offices
- Schools
- Malls
- Touristy Areas (Maybe some lore about the Spanish invading in the 1800's and the architecture being very Spanish... I don't know but I have quite a vision)
- Town Hall
- Parks
- Bike Shops
- Clothing Stores
- Retail Stores
- Movie Theaters
- Car Rentals
- Art Galleries
- Doctors Offices
- Dentists Offices
- Condos, Hotels
- Fire Stations
- Police Stations
- Video Game Stores
- Arcades
- Casinos
- Roller Coaster Parks
- Water Parks
- Pools
- Cafes
- Banks
- Embassy Buildings,
- Pet Shops
- Colleges
- Community Colleges
- High schools
- Spa
- Pizzeria
- Diners
- Hot Dog Stands
- Bars
- Tennis Courts
- Thrift Shops
- Supermarket
- Bowling Alley
- Office Buildings
- Airports
- Zoo
- Aquarium
- Gyms
- Train Stations
- Warehouses

I'll add more and I realize these aren't all shops but...


Edited on 7th Jan 2021 (1:46 PM EST) for organisation. No original text was removed or altered.

**Comment by KadTheHunter** [comment]
I like all of these with the exception of
> (Maybe some lore about the Spanish invading in the 1800's and the architecture being very Spanish... I don't know but I have quite a vision)

Specifically, I like the *lore*  idea (I myself have been thinking on it), but not that specific lore. Currently the downtown is reminiscent of New York City in my opinion, probably looks similar to any other city however (minus the scuzz), though I kind of want to make the terrain and rural areas around the city be similar to Kansas and the prairies. We'll see how it turns out. City before this one (I have no clue what Number and Designation) had lore mirroring RDR2, with the Kad Der Linde gang (yes that's stupid, no I didn't realise at the time).

**Comment by ParadiseMCPE** [comment]
Yeah lore's always important for planning. I just figured any excuse to slap in a beach district with fancy spanish hotels would be good. Disregarding the land-mass and whether or not it's a large island or a wide valley -a city that expands outwards into lesser buildings, neighborhoods, and then country... sounds great.

**Comment by KadTheHunter** [comment]
Oh genius idea, I can add a small mountain range inland with prairies and farms beyond it, and the city (including a coastline, with beaches, factories and docks) on its other side.

**Comment by ParadiseMCPE** [comment]
yeah I also figured if it was coastal you could do some smaller islands connected by bridges to house stuff like the airports (because obviously you wouldn't want tall buildings near where planes are taking off.)

**Comment by ParadiseMCPE** [comment]
also I don't know if this would work for your version of minecraft but this is the best city shader I found 

https://cubed.community/texturepack/

definitely wouldn't work for us pocket edition players.

**Comment by KadTheHunter** [comment]
> https://cubed.community/texturepack/

Aside from the fact that's a Java texture and uses OptiFine, there's currently the issue with W10/BE where pre-RenderDragon shaders refuse to load/work, and most of the post-RenderDragon shaders are RTX only. 
Currently I'm not using any packs (personally or added on to the server) involving changed Textures, or Shading/Lighting. I have a skybox pack that changes the sky/void to a starscape, but that's it.
Most people use Vanilla MC Textures or some very close variant, so I've decided I'll probably stick with Vanilla resources so when I'm building the city, I know it looks good in Vanilla.

**Comment by KadTheHunter** [comment]
I really should make a FAQ section of the README.md, will work on that soon.

Edit: Added in [3ccbd02](https://github.com/KadTheHunter/CityPlans/commit/3ccbd027c907a34f493b8b7ac79ef56fcc7d4093)
